### PR TITLE
UI presenter testing

### PR DESCRIPTION
**Pull request by chubberlisk**

### CODE REVIEW

File: spec/unit/presenter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+  it 'can end the game' do
+    evaluate_game.stop_calls_after = 1
+
+    presenter.execute
+
+    expect(evaluate_game.number_of_calls).to eq(3)
+  end
+
+  it 'can display the end game' do
```
Comment (dushan-madetech) [inline spec/unit/presenter_spec.rb:123]: Tests seem logical and clean

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -2,7 +2,22 @@
 
 require 'zeitwerk'
 
+
+class CustomInflector < Zeitwerk::Inflector
+  def camelize(basename, _abspath)
+    case basename
+    when 'ui'
+      'UI'
+    when 'cli'
```
Comment (craigjbass) [inline spec/spec_helper.rb:11]: Can't seem to see anything in this pull request that relies on CLI ?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -2,7 +2,22 @@
 
 require 'zeitwerk'
 
+
+class CustomInflector < Zeitwerk::Inflector
+  def camelize(basename, _abspath)
+    case basename
+    when 'ui'
```
Comment (craigjbass) [inline spec/spec_helper.rb:9]: If your class names where not abbreviated you would not face this issue.

E.g. `UserInterface::CommandLine` is far more immediately explanatory than `UI::CLI`

File: spec/unit/presenter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,130 @@
+describe UI::Presenter do
```
Comment (craigjbass) [inline spec/unit/presenter_spec.rb:1]: Did you delete your production code before writing these tests?

File: spec/spec_helper.rb (hunk 3 above)
Comment (RobertM905) [reply spec/spec_helper.rb:9]: Very true. We'll do that.

File: spec/unit/presenter_spec.rb (hunk 4 above)
Comment (chubberlisk) [reply spec/unit/presenter_spec.rb:1]: In this case, we did not but we will now rectify this by going back and TDDing the solution.

File: spec/spec_helper.rb (hunk 2 above)
Comment (chubberlisk) [reply spec/spec_helper.rb:11]: Ah, this is true. We thought we might need it. We'll now remove it.

**Pull request closed without merging**